src/api/APIFactory.js

```diff
@@ -206,6 +218,21 @@ class APIFactory {
             delete this.fileCollaboratorsAPI;
         }
 
+        if (this.feedItemsAPI) {
```

Comment:
If this change isn't related to this patchset, can we move it to another PR? It'll help ensure that changes don't get inadvertently lost if we need to revert.